doc/classes/CullInstance.xml

```diff
@@ -15,10 +15,6 @@
 	<methods>
 	</methods>
 	<members>
-		<member name="allow_merging" type="bool" setter="set_allow_merging" getter="get_allow_merging" default="true">
-			This allows fine control over the mesh merging feature in the [RoomManager].
-			Setting this option to [code]false[/code] can be used to prevent an instance being merged.
-		</member>
```

Comment:
This breaks the API technically, and while we occasionally do it in minor releases (e.g. the fully rewritten navigation code), it's best if we can avoid it. So you could keep the property and mark it as deprecated in the docs (and by what new system), and ideally handle converting it to the new system (I guess the Spatial merging mode).

You can handle backward compatibility by overriding `Object::_set` and `Object::_get` and handling what should be done with the deprecated property name.

Reply:
Ok I'll have a look and see if we can stay backward compatible. :+1:

Reply:
It now respects the old flag (for `RoomManager`) and there is a note that it is deprecated in the docs.